src/client/pages/EmailSent.tsx

```diff
@@ -47,17 +71,26 @@ export const EmailSent = ({
           <PageHeader>Check your email inbox</PageHeader>
           <PageBody>
             {email ? (
-              <PageBodyText>We’ve sent an email to {email}.</PageBodyText>
+              <PageBodyText>
+                We’ve sent an email to <b>{email}</b>.
```

Comment:
Text bolded and subtitle changed to match Figma design 
![Screenshot 2021-10-05 at 11 26 48](https://user-images.githubusercontent.com/1771189/136006556-48fbd5aa-75e3-456a-aa22-edb27aa7b046.png)

Reply:
Also removed some `<br/>`s to match the design spec margins